spec/lib/solidus_jwt/devise_strategies/refresh_token_spec.rb

```diff
@@ -13,13 +13,11 @@
     }
   end
 
-  let(:headers) { {} }
-  let(:user) { FactoryBot.create(:user, password: password) }
-  let(:password) { 'secret' }
+  let(:user) { FactoryBot.create(:user, password: 'secret') }
   let(:token) { user.auth_tokens.create! }
 
   before do
-    allow(request).to receive(:headers).and_return(:headers)
```

Comment:
I don't understand this change, is `:headers` the same as `{}` in this context? Is this suggested by Rubocop?

Reply:
MultipleMemoizedHelpers
there were 7 in this spec so I had to try and reduce 2
Standardrb sounds pretty nice right now in place of Rubocop eh? 😄